docs/engineering-principles.md

```diff
@@ -0,0 +1,48 @@
+# Engineering Principles
+Engineering principles define the core tenets and beliefs that the engineering team and all contributors are expected to keep front of mind. These tenets are meant to reflect the values of the organization as well as the philosophy upon which the software was founded. Principles typically come in the form of punchy one line statements, but a great deal of depth can be written on each one to further inform of the intent of the statement.
+
+## Core Engineering Values
+These one word statements are used to not only capture the qualities we strive for in our software, but also prioritizes those values. Keep in mind that being the last value in this list does not imply it is of no importance, it is still more important than all of the other qualities not mentioned here. 
+1. Security
+2. Stability
+3. Safety
```

Comment:
Curious how this differs from "security". Depending on the definition, one might encompass the other.

Maybe these should all have definitions anyway

Reply:
Probably yeah. I'll work on definitions and condense the two mentioned if i can't articulate the difference.

Reply:
@Gudahtt i'm going to remove safety, for now, and put off writing descriptions for these to a new PR